source/install-from-source.txt

```diff
@@ -0,0 +1,373 @@
+        
+        ## Download using wget:
+        $ wget "https://github.com/mongodb/mongo-c-driver/archive/refs/tags/$VERSION.tar.gz" \
+             --output-document="mongo-c-driver-$VERSION.tar.gz"
+        ## Extract using tar:
+        $ tar xf "mongo-c-driver-$VERSION.tar.gz"
+
+   .. tab:: Using ``curl`` + ``tar``
+      :tabid: curl-tar
+
+      .. code-block:: none
+        
+        ## Using curl:
+        $ curl "https://github.com/mongodb/mongo-c-driver/archive/refs/tags/$VERSION.tar.gz" \
+             --output="mongo-c-driver-$VERSION.tar.gz"
+        ## Extract using tar:
+        $ tar xf "mongo-c-driver-$VERSION.tar.gz"
+
+   .. tab:: PowerShell
+      :tabid: powershell
+
+      .. code-block:: none
+
+        ## Use Invoke-WebRequest:
+        PS> $url = "https://github.com/mongodb/mongo-c-driver/archive/refs/tags/$VERSION.zip"
+        PS> $file = "mongo-c-driver-$VERSION.zip"
+        PS> Invoke-WebRequest -UseBasicParsing -Uri $url -OutFile $file
+        ## Extract using Expand-Archive:
+        PS> Expand-Archive mongo-c-driver-$VERSION.zip
+
+The above commands will create a new directory ``mongo-c-driver-$VERSION`` within
+the directory in which you ran the ``tar`` / ``Expand-Archive`` command (**note**:
+PowerShell will create an additional intermediate subdirectory of the same
+name). This directory is the root of the driver source tree (which we refer to
+as ``$SOURCE`` in these documents). The ``$SOURCE`` directory should contain the
+top-level ``CMakeLists.txt`` file.
+
+
+Obtaining Prerequisites
+-----------------------
+
+In order to build the project, a few prerequisites need to be available.
+
+Both ``libmongoc`` and ``libbson`` projects use `CMake <https://cmake.org>`__ for build configuration.
+
+.. note::
+
+   It is *highly recommended* -- but not *required* -- that you download the
+   latest stable CMake available for your platform.
+
+.. tip:: Getting the Latest CMake
+   
+   A new stable release of CMake can be obtained from
+   `the CMake downloads page <https://cmake.org/download/#latest>`__.
+
+   For Windows and macOS, simply download the CMake ``.msi`` / ``.dmg`` (not the
+   ``.zip`` / ``.tar.gz``) and use it to install CMake.
+
+   On Linux, download the self-extracting shell script (ending with ``.sh``) and
+   execute it using the ``sh`` utility, passing the appropriate arguments to
+   perform the install. For example, with the CMake 3.27.0 on the ``x86_64``
+   platform, the following command can be used on the
+   ``cmake-3.27.0-linux-x86_64.sh`` script:
+
+   .. code-block:: none
+
+      $ sh cmake-3.27.0-linux-x86_64.sh --prefix="$HOME/.local" --exclude-subdir --skip-license
+
+   Assuming that ``$HOME/.local/bin`` is on your ``$PATH`` list, the ``cmake`` command
+   for 3.27.0 will then become available.
+   
+   The ``--help`` option can be passed to the shell script for more information.
+
+For the remainder of this page, it will be assumed that ``cmake`` is available as
```

Comment:
I: I'm not as anti-passive-voice as the [style guide](https://www.mongodb.com/docs/meta/style-guide/writing/use-active-voice/), but it seems easy to avoid here

```suggestion
This page assumes that ``cmake`` is available as
```